### DISCUSSION TITLE
how to make quality pull request in hactoberfest?

### DISCUSSION THREAD

**Discussion by NirmalSilwal** [EddieHubCommunity/support#248] [Q&A]
Hi open source contributor,

The [registration has opened for hackoberfest]( https://hacktoberfest.digitalocean.com) this year. I am going to participate for the first time this year. Can I get some suggestions to make quality PR being a beginner?

How to stand out in **hactoberfest 2020**? 

Thanks

**Comment by AllanRegush** [comment]
I'd make a issue that is well written and has a screenshot if applicable.
Then make a PR with closes $yourIssueNumber. Describe what you changed add a screenshot if applicable.

All you really have to worry about it making sure your adding value to the project.

**Reply by eddiejaoude** [reply]
Good question @NirmalSilwal and great tips @AllanRegush 👍 

I think I might try and do a video on this - thank you for the idea both of you 🤓

**Reply by NirmalSilwal** [reply]
Thank you @AllanRegush and @eddiejaoude for putting your views.

Absolutely @eddiejaoude brief video would be helpful in this case.  Let's hear what other contributor have suggestions!
